betterforms/multiform.py

```diff
@@ -38,6 +39,7 @@ def __init__(self, data=None, files=None, *args, **kwargs):
         if self.initials is None:
             self.initials = {}
         self.forms = OrderedDict()
+        self.crossform_errors = []
```

Comment:
Maybe this should be a private API `_crossform_errors`, and be merged later with the `property(errors)` ? What do you think?